Quorum/config.py

```diff
@@ -1,9 +1,12 @@
 import os
 import shutil
 from pathlib import Path
+import dotenv
 
 import Quorum.utils.pretty_printer as pp
 
+dotenv.load_dotenv(override=True)
```

Comment:
Thinking out loud, should we alert the user on override cases?

Reply:
Hmm i guess its a good idea

Reply:
Not sure how to do it tbh

Reply:
Probably something like - 
for a in .env[vars]:
if a in os.getenviron() : print(something)

Reply:
Its not straight forward, but this should work, just a solution from my head without investigating too much

Reply:
Done